prometheus-jmx-exporter/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM eclipse-temurin:11.0.12_7-jre-focal@sha256:23d5cad605d1d2ef79098016397e1c8fb993d28df79cf41c40a6904ae779f4ec
+FROM eclipse-temurin:11.0.13_8-jdk-focal@sha256:e36def1e27583cce3786e17b3736b996b5536c01515f2863984823451710c248
```

Comment:
Would it make sense to use `eclipse-temurin:11-jdk-focal` - then we can just rebuild without having to update the tag? Also it seems to be `11.0.14.1+1`.